modules/api/lambda/package.json

```diff
@@ -23,7 +23,6 @@
     "lint": "make lint",
     "start": "make start",
     "test": "make test",
-    "test:acceptance": "make test-acceptance",
```

Comment:
Why did you remove the acceptance tests script?

Reply:
I didn’t. Original branch again. Will rollback this too.

Reply:
Oh my, sorry for the confusion. I haven't been working on this project for quite some time now. All good. It's fine.